docs/src/app/components/pages/components/app-bar.jsx

```diff
@@ -113,6 +114,16 @@ export default class AppBarPage extends React.Component {
         name="AppBar"
         desc={this.desc}
         componentInfo={this.componentInfo}>
+
+        <Paper style = {{marginBottom: '22px'}}>
+          <CodeBlock>
+          {
+            '//Import statement:\nconst AppBar = require(\'material-ui/lib/app-bar\');\n\n' +
+            '//See material-ui/lib/index.js for more\n'
```

Comment:
Of course, we could. But it just seemed like a small piece of code that I could easily add to each page. Also, if I were to create a component for this, passing in data for the import statements as props might end up being more lines of code than this.